### ISSUE TITLE
How to enable for all source files?

### ISSUE THREAD

**Issue by inancgumus** [braver/typewriter#15]
~~There is no settings option in the installed package.~~

**Comment by inancgumus** [comment]
I set the configuration like this but it does not care:

``` js
atom.config.set('typewriter.scopes', 'source.gfm,source.js,source.html')
```

**Comment by inancgumus** [comment]
It works now. But,

Interesting, I've found the grammars like this:

``` js
Object.keys(atom.grammars.grammarsByScopeName).filter(function(s){ return /.+js.*/.test(s); }).sort().join('\n')
```

It returned:

```
source.coffee.jsx
source.js
source.js.jsx
source.js.rails source.js.jquery
source.js.regexp
source.js.regexp.replacement
source.json
source.ruby.rails.rjs
text.html.jsp
```

Then, I've changed the config to scopes: 

``` json
  typewriter:
    scopes: "source.gfm,source.js,source.js.jsx,source.js.jquery,source.json,text.html.basic,text.html"
```

**How is it possible to apply it for all types of files?**

**Comment by inancgumus** [comment]
With this command I've enabled it for all types of files. But, I think this could be a setting in the plugin.

``` js
atom.config.set(
  'typewriter.scopes', 
  Object.keys(atom.grammars.grammarsByScopeName).sort().join(',')
)
```

**Comment by braver** [comment]
fixed in [0.6.0](https://github.com/braver/typewriter/releases/tag/v0.6.0)
